lib/ui-application/server.ioc-container.js

```diff
@@ -29,7 +29,7 @@ export function createServerIocContainer({ baseConfigPath, configPath, assetsMan
     CONFIG_PATH: configPath,
     MANIFEST_PATH: assetsManifestPath,
     configReader: iocClass(ConfigurationReader),
-    manifestManager: iocClass(AssetsManifestManager),
+    manifestManager: iocFactory(() => new AssetsManifestManager(require(assetsManifestPath))),
```

Comment:
I think we need to move loading (requiring) of `assetsManifestPath` to high-level function (caller of `createServerIocContainer`) in order to make this function is pure with such side effects